src/mvt_sql.py

```diff
@@ -148,26 +148,12 @@ def get_vectors(minzoom, maxzoom, x, y, style, vec, extent):
                 add = "(" + add + ")"
                 adp.append(add)
 
-    if "name:en" in column_names:
-        mapped_cols[
-            "name:en"
-        ] = """coalesce(
-        tags->'name:en',
-        tags->'int_name',
-        replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace
```

Comment:
can we keep this?

Reply:
yes, but then `name:lang` in tiles always be completed with `name` so coalescing on client side won't make sense. it would work only if `name:lang` is completed with names where replace actually replaced something. am i wrong here?

Reply:
no, just the last line should be changed name -> name:ru and then you will still have a lot of places to coalesce in.